Customizing/global/plugins/Services/Cron/CronHook/EffectivenessAnalysisReminder/classes/ilEffectivenessAnalysisReminderDB.php

```diff
@@ -58,64 +77,46 @@ protected function createTable() {
 	 *
 	 * @param int 		$crs_id
 	 * @param int 		$superior_id
-	 * @param string 		$type
+	 * @param string 	$type
+	 * @param int[]		$user_ids
 	 */
-	public function reminderSend($crs_id, $superior_id, $type) {
-		$values = array("crs_id" => array("integer", $crs_id)
+	public function logMailSend($crs_id, $superior_id, $type, $user_ids) {
+		$user_ids = base64_encode(serialize($user_ids));
```

Comment:
Ich glaube hier wäre ein `implode(",", $user_ids)` besser. Dann sieht man in der Tabelle noch, was drinsteht. Warum sind die ganzes user_ids eigentlich in einer Zelle gespeichert, statt pro user_id eine neue Zeile?